docs/_static/monitoring/v1.yaml

```diff
@@ -0,0 +1,65 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Node Monitoring API spec
+  description: API specification for monitoring of the nuts node
+  version: 1.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /status:
+    get:
+      summary: "healthcheck to confirm node is up and running"
+      description: |
+        Check if the node is up and running.
+      operationId: getStatus
+      tags:
+        - status
+      responses:
+        200:
```

Comment:
and otherwise?

Reply:
We only have a 200 set for this (and all other monitoring) endpoints. Not sure what else you would like to see here?